params/config.go

```diff
@@ -169,6 +172,11 @@ func (c *ChainConfig) String() string {
 	)
 }
 
+// IsEWASM returns whether num represents a block number after the EWASM fork
+func (c *ChainConfig) IsEWASM(num *big.Int) bool {
+	return isForked(c.EWASMBlock, num)
+}
```

Comment:
There is a nice progression of forks defined below starting from frontier, going towards forward in time. Why did you put this in front of everything else?

Reply:
aesthetics ;)

It should be fixed now